integrations/doubleclick-floodlight/lib/index.js

```diff
@@ -28,6 +41,18 @@ var Floodlight = module.exports = integration('DoubleClick Floodlight')
  */
 
 Floodlight.prototype.initialize = function() {
+  // In the initialize method:
+  // Check if we should load the DoubleClick ID pixel (and only proceed if we haven't already done so).
+  if (this.options.getDoubleClickId && this.options.googleNetworkId) {
+    // Load the doubleclick pixel.
+    this.load('doubleclick id', {
+      googleNetworkId: this.options.googleNetworkId,
+      segmentWriteKey: this.options.segmentWriteKey,
+      // TODO: handle userId being nulls/undefined.
```

Comment:
Yeah, currently we end up with `user_id=null` in img src which is confusing for idsync-service. I'll filter this value out for now.

Reply:
Hey @andreiko - should we only fire the pixel if `userId` exists?

Reply:
Can we just remove the `user_id` parameter from the pixel url in that case?

Reply:
We can but is it not required to actually facilitate the match or can you do it with `anonymousId`?

Reply:
yeah, it would work with just the `anonymousId`